src/Common/src/System/Diagnostics/Debug.cs

```diff
@@ -97,7 +97,7 @@ public static void Assert(bool condition, string message, string detailMessage)
 
                 try
                 {
-                    stackTrace = Environment.StackTrace;
+                    stackTrace = Internal.Runtime.Augments.EnvironmentAugments.StackTrace;
```

Comment:
Is this because System.Diagnostics.Debug.dll can't depend on System.Runtime.Extensions.dll?